universal-application-tool-0.0.1/app/views/WebJarJsPaths.java

```diff
@@ -0,0 +1,30 @@
+package views;
+
+import java.util.Optional;
+
+/** Enum for holding the paths for web JARs needed throughout the code base. */
+public enum WebJarJsPaths {
+  AZURE_STORAGE_BLOB("lib/azure__storage-blob/browser/azure-storage-blob.min.js"),
+  ;
+
+  private final String webJarPath;
+
+  WebJarJsPaths(String webJarPath) {
+    this.webJarPath = webJarPath;
+  }
+
+  /** Returns the enum associated with the provided String value. */
+  public static Optional<WebJarJsPaths> forString(String string) {
```

Comment:
question (blocking): what is this method needed for?

Reply:
Removed the method since we've switched over to a static class